relay-server/src/services/processor/replay.rs

```diff
@@ -262,13 +264,42 @@ fn handle_replay_recording_item(
     })
     .map(Into::into)
     .map_err(|error| {
+        match &error {
+            relay_replays::recording::ParseRecordingError::Compression(e) => {
+                // 20k errors per day at 0.1% sample rate == 20 logs per day
+                if rand::thread_rng().gen_range(1..=1000) == 1 {
+                    relay_log::warn!(
+                        error = e as &dyn Error,
+                        event_id = ?config.event_id,
+                        project_id = config.project_id.map(|v| v.value()),
+                        organization_id = config.organization_id.map(|o| o.value()),
+                        encoded_payload = BASE64_STANDARD.encode(payload),
```

Comment:
As discussed let's use an attachment for this:

Example:
```rs
                    relay_log::with_scope(
                        move |scope| {
                            // truncate the payload to basically 200KiB, just in case
                            payload.truncate(200_000);
                            scope.add_attachment(relay_log::protocol::Attachment {
                                buffer: payload.into(),
                                filename: "payload.json".to_owned(),
                                content_type: Some("application/json".to_owned()),
                                ty: None,
                            })
                        },
                        || {
                            relay_log::error!(
                                error = &error as &dyn Error,
                                "my error"
                            )
                        },
                    );
```